### PR TITLE
Added loader

### PR DESCRIPTION
**Pull request by aswinzz**

### CODE REVIEW

**Comment by gauthamzz** [comment]
Thanks

File: static/css/loader.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+body{
+  background: #3498db;
+  margin: 150px 0;
+  text-align: center;
+  color: #fff;
+  font-family: 'Oswald', sans-serif;
+}
+.content{
+  letter-spacing: 2px;
+  text-transform: uppercase;
```
Comment (himanshub16) [inline static/css/loader.css]: This has caused all the text in the post to transform to uppercase.
The transform should be limited to the text in the loader only.
Please fix it ASAP.

**Comment by himanshub16** [comment]
Please make another PR to fix this.

**Comment by aswinzz** [comment]
ok sir

**Pull request merged**